src/osd/OSDMap.cc

```diff
@@ -1807,12 +1807,6 @@ void OSDMap::clean_temps(CephContext *cct,
       pending_inc->new_pg_temp[pg.first].clear();
       continue;
     }
-    if (!nextmap.pg_exists(pg.first)) {
```

Comment:
@ljflores 
I don't know how to explain this change, it is not something I did, and I don't know how to see git blame on these lines (since they are not part of my file), but if this is a new addition to the file, we should add this to this PR (or rebase)

Reply:
@JoshSalomon I would rebase:

`git pull <remote> <branch> --rebase`, where "remote" refers to the actual ceph repository, not your forked version. In my  case, it would be `git pull ceph <branch> --rebase`.

You can use `git remote -v` to check what you named the `git@github.com:ceph/ceph.git` repository.